Gruntfile.js

```diff
@@ -37,6 +37,12 @@ module.exports = (grunt) => {
 
     grunt.initConfig({
         copy: {
+            assets: { // assets: images, htmls, etc.
```

Comment:
Is the intention to move the other UIs into this `assets` dir? Like certificate and login?

Reply:
You are right, yes, I was going to move the rest of the assets into this folder, but in a separate PR 👍